docker/Makefile

```diff
@@ -11,7 +11,21 @@ ifndef version
 endif
 
 build: .TEST
```

Comment:
Hmm .. I just realized that that ".TEST" target here is no used is it?  `$(error)` can just be in column zero and fire when the Makefile is parsed.    No need to special ".TEST" rule I think.

Not related to this PR though .

Reply:
Not sure if I understand, the `.TEST` is meant to verify 'arguments' and make sure make is feed with `version` variable:
```
make: Entering directory '/home/trzeci/Projects/emsdk/docker'
Makefile:10: *** argument 'version' is not set. Please call `make version=SOME_VERSION ...`.  Stop.
make: Leaving directory '/home/trzeci/Projects/emsdk/docker'
```